addons/maintenance/migrations/13.0.1.0/post-migration.py

```diff
@@ -0,0 +1,19 @@
+from openupgradelib import openupgrade
+
+
+def replace_field_active_for_archive(env):
```

Comment:
đổi lại tên hàm cho chuẩn ý nghĩa
```suggestion
def merge_field_active_to_archive(env):
```

Reply:
done a nhé.